ghcup-vanilla-0.0.8.yaml

```diff
@@ -4794,14 +4794,166 @@ ghcupDownloads:
         dlSubdir: ghc-9.10.1
         dlUri: https://downloads.haskell.org/~ghc/9.10.1/ghc-9.10.1-src.tar.xz
       viTags:
-      - Recommended
-      - Latest
       - base-4.20.0.0
       viTestDL:
         dlHash: bc03b1ec1f843bb5471003678f3d8de6d6625ee6a67db8c6cc2efaaea55206e1
         dlSubdir: ghc-9.10.1/testsuite
         dlUri: https://downloads.haskell.org/~ghc/9.10.1/ghc-9.10.1-testsuite.tar.xz
-
+    9.12.1:
+      viArch:
+        A_32:
+          Linux_Debian:
+            ( >= 10 && < 12 ): &ghc9121-i386-linux-deb10
+              dlHash: 1e04fb8e6f177a8d72f99b6f7e60056f5531a501a16a29355c25f629a1662d66
+              dlOutput: ghc-9.12.1-i386-linux-deb10.tar.xz
```

Comment:
Is there any harm to keeping it for now? The metadata fragment is now almost entirely machine generated, I can open an issue/MR to GHC to avoid including this field in future releases.

Reply:
It should be cleaned up. The yaml is supposed to be human readable.

It makes it hard to see the spots where problematic bindist URIs exist.